tests/conftest.py

```diff
@@ -1,96 +0,0 @@
-import os
-import tempfile
-
-import boto3
-from botocore.stub import Stubber
-from pytest import fixture
-
-from blobby import blob, client
-
-
-@fixture
-def s3():
-    s3 = boto3.resource('s3')
-    yield s3
-
-
-@fixture
-def stubber(s3):
-    with Stubber(s3.meta.client) as stubber:
-        yield stubber
-        stubber.assert_no_pending_responses()
-
-
-@fixture
-def bucket(s3):
-    bucket = s3.Bucket('mock-bucket')
-    yield bucket
-
-
-@fixture
-def reader(bucket):
-    reader = blob.BlobReader(bucket, 'test')
-    yield reader
-
-
-@fixture
-def writer(bucket):
-    writer = blob.BlobWriter(bucket)
-    yield writer
-    try:
-        writer.cancel()
-    except ValueError:
-        pass
-
-
-class BlobReaderMock:
-    def __init__(self, *args, **kwargs):
-        pass
-
-    def read(self):
-        return b'mocked'
-
-
-@fixture
-def blob_reader_mock(monkeypatch):
-    monkeypatch.setattr(blob, 'BlobReader', BlobReaderMock)
-
-
-class BlobWriterMock:
```

Comment:
Might be worth defining an `ABC` or using mypy protocols https://mypy.readthedocs.io/en/stable/protocols.html if we are happy to run on only 3.8. for the `BlobWriter` and `BlobReader` to ensure that the mocks and real implementations don't get out of sync.